asab/library/providers/zookeeper.py

```diff
@@ -314,7 +315,14 @@ def build_path(self, path):
 
 
 	async def subscribe(self, path):
```

Comment:
add optional argument `target` with possible values:
- `"global"` (default) to watch global target changes
- `"tenant"` to watch ANY tenant changes
- `("tenant", TENANT_ID)` to watch a specific tenant changes